Detectors/MUON/MCH/DigitFiltering/src/digits-filtering-workflow.cxx

```diff
@@ -12,7 +12,7 @@
 #include "Framework/CallbackService.h"
 #include "Framework/ControlService.h"
 #include "Framework/Task.h"
-#include "DigitFilteringSpec.h"
+#include "MCHDigitFiltering/DigitFilteringSpec.h"
```

Comment:
You also need the add configKeyValues to the workflow options and   o2::conf::ConfigurableParam::updateFromString(cc.options().get<std::string>("configKeyValues"));
to be able to change the parameters from the command line.